test/hotel_block_test.rb

```diff
@@ -0,0 +1,89 @@
+require_relative "test_helper"
+
+describe "HotelBlock" do
+  let(:date_range) {
+    date_range = Hotel::DateRange.new(Date.new(2020,5,3), Date.new(2020,5,5))
+  }
+  let(:reservations) {
+    rooms.each do |room|
+      room.reservations << Hotel::Reservation.new(date_range, 2, room.number, room.cost, 1, true)
+    end
+  }
+  let(:rooms) {
+    room1 = Hotel::Room.new(1, 200)
+    room2 = Hotel::Room.new(2, 200)
+    room3 = Hotel::Room.new(3, 200)
+    rooms = [room1, room2, room3]
+  }
```

Comment:
the let syntax is super super super weird. It wants to assign the local variable to the last line evaluated in the let block.

This is all to say, in this block that defines a variable named `rooms` (determined by the line `let(:rooms)`), you technically don't need the `rooms =` part on line 16. This will help get rid of a warning you have when you run `rake` that says `.../hotel/test/hotel_manager_test.rb:16: warning: assigned but unused variable - rooms`

When you run rake, if you see a bunch of these warnings about `assigned but unused variable`s, and it's about a variable in a `let` block, try deleting the `var_name =` part on the last line. Ruby is trying to say that that's redundant with the `let` block syntax.